distributed/scheduler.py

```diff
@@ -7920,6 +7920,62 @@ def adaptive_target(self, comm=None, target_duration=None):
             to_close = self.workers_to_close()
             return len(parent._workers_dv) - len(to_close)
 
+    def acquire_replicas(self, addr: str, keys: list, *, stimulus_id: str):
```

Comment:
Should this be called something like `request_acquire_replicas` or `request_worker_acquire_replicas`, since it may fail/be ignored, and returns before any work is actually done? No strong thoughts.

Reply:
changed